app/src/main/kotlin/org/fossify/calendar/helpers/IcsImporter.kt

```diff
@@ -26,6 +26,7 @@ class IcsImporter(val activity: SimpleActivity) {
 
     private var curStart = -1L
     private var curEnd = -1L
+    private var duration = 0
     private var curTitle = ""
```

Comment:
But do try to blend in with the existing code style (even if it's outdated). The variable should have been named `curDuration` instead of just `duration`.